pkg/server/updates_test.go

```diff
@@ -161,6 +161,9 @@ func TestReportUsage(t *testing.T) {
 			t.Fatalf("error applying zone config %q to %q: %v", cmd.config, cmd.resource, err)
 		}
 	}
+	if _, err := db.Exec(`INSERT INTO system.zones (id, config) VALUES (10000, null)`); err != nil {
```

Comment:
We should also add a test for a null ID while we're here.

Reply:
I don't think that's possible: `id     INT PRIMARY KEY,`

Reply:
Duh, of course. Nevermind.